art-Service_Provisioning_Using_an_Orchestration_Template/miq/index.adoc

```diff
@@ -1,4 +1,4 @@
-= Service Provisioning in CloudForms Using an Orchestration Template
+= Service Provisioning in ManageIQ Using an Orchestration Template
```

Comment:
Should this one be `{product.title}` ?

Reply:
Not necessarily, since it is in the `miq` subdirectory.